<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Multiplayer</class>
 <widget class="QDialog" name="Multiplayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1035</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>999</width>
     <height>625</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>627</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Imagenes.qrc">
     <normaloff>:/botonMenu.png</normaloff>:/botonMenu.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>150</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="ControJ1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>627</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Imagenes.qrc">
     <normaloff>:/botonControles - copia.png</normaloff>:/botonControles - copia.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>150</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="ControlJ2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>627</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Imagenes.qrc">
     <normaloff>:/botonControles - copia.png</normaloff>:/botonControles - copia.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>150</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>630</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Vidas J1:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="vidasJ1">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>630</y>
     <width>112</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Vidas J2:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="vidasJ2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
